mypy/checkmember.py

```diff
@@ -559,6 +562,15 @@ def analyze_class_attribute_access(itype: Instance,
             msg.fail(messages.GENERIC_INSTANCE_VAR_CLASS_ACCESS, context)
         is_classmethod = ((is_decorated and cast(Decorator, node.node).func.is_class)
                           or (isinstance(node.node, FuncBase) and node.node.is_class))
+        if (is_classmethod and isinstance(itype, Instance)
```

Comment:
`itype` should be _always_ and `Instance` (see annotation). Have you found a test where it is not?

Reply:
@ilevkivskyi 
It seems to be not necessary as you say. I will remove it.